framework/ui/internal/uiengine.cpp

```diff
@@ -95,6 +96,30 @@ void UiEngine::setup(QQmlEngine* e)
 #endif
 
     m_engine->addImportPath(":/qml");
+
+#ifdef QML_LOAD_FROM_SOURCE
+    for (const QString& path : m_sourceImportPaths) {
+        m_engine->addImportPath(path);
+    }
+#endif
+}
+
+void UiEngine::addSourceImportPath(const QString& path)
+{
+#ifdef QML_LOAD_FROM_SOURCE
+    LOGD() << path;
```

Comment:
I think you can remove it

Reply:
Better keep, because:
1. Show what load qml from source enabled
2. Show path to qml
3. Show what path set in the module project file